ibis/backends/clickhouse/__init__.py

```diff
@@ -29,6 +29,8 @@
 
 class Backend(BaseSQLBackend):
     name = 'clickhouse'
+    # for now map clickhouse to mysql so that _something_ works
+    _sqlglot_dialect = "mysql"
```

Comment:
clickhouse exists as a dialect in sqlglot now, so `"clickhouse"` should work

Reply:
Good question, I don't think so, but feel free to do that here. Bumping the minimum version to the latest version `9.0.5` is fine.